package.json

```diff
@@ -148,7 +148,7 @@
     "identity-obj-proxy": "^3.0.0",
     "image-webpack-loader": "3.4.2",
     "imports-loader": "0.7.1",
-    "jest": "^20.0.4",
+    "jest": "^22.2.1",
```

Comment:
can we take off the `^` caret to pin it. I've gotten bit by minor upgrades breaking npm packages when making prod builds many times lol.